book/website/history.md

```diff
@@ -0,0 +1,43 @@
+A GitHub repository was made for the project, and internal processes decided in how acknowledge of authorship works within the project (i.e. installation of Welcome bot, All-Contributors bot) and collaborative spaces and events to contribute to the guides were created (i.e. Collaboration Cafe, Book Dash).
+In-person workshops were given in person on tools aligned with the project: particulary Binder and JupterBook, and the project was presented in many conferences across the UK and internationally.
+
+## Development and Growth (2019-2021)
+
+With the addition of Dr. Malvika Sharan in late 2019, The Turing Way expanded into five guides.
+As a co-founder of Open Life Science (which uses many Mozilla OL frameworks), Malvika helped to incubate what are now key partnerships for The Turing Way.
+This period corresponded with the COVID-19 pandemic lockdowns, which transitioned many processes of contribution, collaboration, and documentation into remote ones.
+
+In part due to this, The Turing Way was able to expand greatly, from one guide on Reproducibility into five different guides related to project design, communication, collaboration, and research ethics.
+A global (and multilingual) community began to engage with The Turing Way, and the process of translation/localisation began informally.
+A Community Handbook was also established to document best practices that could be used by other communities.
+The Fireside Chat series was established in late 2021, facilitating shared spaces with networks and allies from across the open science ecosystem.
+The project received a few awards & citations in the 2021 London Mayor report, HiddenREF Awards, and Goldacre Report.
+The Book Dash Planning Committe was established to plan the community's flagship collaboration event.
+
+## Maintenance & Sustainability (2022-Ongoing)
+
+In 2022, Malvika and Kirstie became co-leads of *The Turing Way*, and welcomed Anne Lee Steele as the new community manager.
+As the number of contributors to the project numbers has grown to over 400 people, The Turing Way has also developed into a kind of ["community of communities"](https://ben.balter.com/2019/07/18/a-community-of-communities-oscon-2019/) with many ongoing projects developing alongside the expansion of the guides, demonstrating the need for internal systems that enable the sustainability and maintenance of the project more broadly.
+Initial governance work began in 2022, with hybrid hubs hosting events across multiple time zones and working groups developed to formalise localisation and translation, reviewing & editing, training and mentorship, accessibility, and infrastructure within the project.
+
+2023 saw the incubation of The Turing Way Practioner's Hub that aims to bring The Turing Way into the hands and ways of working of industry leaders, as well as a transition into a separate Github organisation for The Turing Way project.
```

Comment:
Maybe rewrite the next two sentences with The Turing Way as the main objective instead of the year. Now it looks like we're passively guided by time instead of via active thought/actions!